src/hal/i_components/joyhandle.icomp

```diff
@@ -17,11 +17,11 @@
 component joyhandle "sets nonlinear joypad movements, deadbands and scales";
 pin in float in;
 pin out float out;
-param rw float power = 2.0;
-param rw float deadband = 0.;
-param rw float scale = 1.;
-param rw float offset = 0.;
-param rw bit inverse = 0;
+pin io float power = 2.0;
+pin io float deadband = 0.;
+pin io float scale = 1.;
```

Comment:
in this file are various instances of 1. and 0. usage. The should be corrected with 1.0 and 0.0. Furthermore, there are more numbers where .0 should be added to numbers to indicate floating point operations.